packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/StakeInANutshellModal/StakeInANutshellModal.tsx

```diff
@@ -109,9 +109,7 @@ export const StakeInANutshellModal = ({ onCancel, flow }: StakeInANutshellModalP
     const { validatorWithdrawTime, validatorExitTime } = useSelector(state =>
         selectValidatorsQueueData(state, account?.symbol),
     );
-    const apy = useSelector((state: StakeRootState) =>
-        selectPoolStatsApyData(state, account?.symbol),
-    );
+    const apy = useSelector((state: StakeRootState) => selectPoolStatsApyData(state, account));
```

Comment:
```suggestion
    const apy = useSelector(state => selectPoolStatsApyData(state, account));
```

No need to specify `StakeRootState` here.

Reply:
fixed everywhere where I found it https://github.com/trezor/trezor-suite/commit/45eff808cc696e01a6581160b316bdd6689a00e9